ui-frontend/packages/catalog-ui-search/src/main/webapp/index.html

```diff
@@ -29,19 +29,98 @@
 </head>
 
 <body>
-    <header></header>
+    <header>
+        <style type=text/css>
+            #incompatible-browser-bg {
+                position: fixed;
+                top: 0;
+                left: 0;
+                text-align: center;
+                line-height: 30px;
+                width: 100%;
+                height: 100%;
+                background-color: #0b1821;
+                overflow: auto;
+                font-family: 'Open Sans', arial, sans-serif;
+            }
+
+            #incompatible-browser {
+                display: table-cell;
+                vertical-align: middle;
+                padding: 20px;
+            }
+
+            #incompatible-browser li {
+                width: 176px;
+                display: inline-block;
+                vertical-align: top;
+                margin: 10px;
+                zoom: 1;
+                *display: inline;
```

Comment:
Is there supposed to be a * in front of display?

Reply:
Yup, it's a hack to get the list to look like `inline-block` in IE 6 and 7, because `inline-block` doesn't seem to be properly supported.